cfme/fixtures/pytest_selenium.py

```diff
@@ -889,7 +889,7 @@ def checkbox(loc, set_to=False):
             selected = el.is_selected()
 
         if selected is not set_to:
-            logger.debug("Setting checkbox %s to %s", str(loc), str(set_to))
+            logger.debug("Setting checkbox to %s", str(set_to))
```

Comment:
This is to fix : 
if selected is not set_to:
>               logger.debug("Setting checkbox %s to %s", str(loc), str(set_to))
E               TypeError: __str__ returned non-string (type WebElement)